### PR TITLE
Update to 5.0 RC1

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

File: Hosted/Client/Startup.cs [hunk 1]
```diff
@@ -2,17 +2,20 @@
...
-using Microsoft.AspNetCore.Components.Builder;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using System.Net.Http;
+using System.Threading.Tasks;
 
 namespace BlazorGrpcHosted.Client
 {
-    public class Startup
+    public class Program
```
Comment (JamesNK) [inline Hosted/Client/Startup.cs]: Program class is in Startup.cs

File: Hosted/Client/wwwroot/index.html [hunk 2]
```diff
@@ -11,7 +11,7 @@
 </head>
 
 <body>
-    <app>Loading...</app>
+    <div id="app">Loading...</div>
```
Comment (JamesNK) [inline Hosted/Client/wwwroot/index.html:14]: What is the reason for this?

I have a Blazor sample at https://github.com/grpc/grpc-dotnet/tree/master/examples/Blazor. Should it be updated to match?

File: Standalone/BlazorGrpcStandalone/Startup.cs [hunk 3]
```diff
@@ -1,17 +1,21 @@
...
+using Microsoft.AspNetCore.Components;
+using Microsoft.AspNetCore.Components.WebAssembly.Hosting;
 using Microsoft.Extensions.DependencyInjection;
 using System.Net.Http;
+using System.Threading.Tasks;
 
 namespace BlazorGrpcStandalone
 {
-    public class Startup
+    public class Program
```
Comment (JamesNK) [inline Standalone/BlazorGrpcStandalone/Startup.cs]: Program in Startup.cs again

**Review by JamesNK — APPROVED** [review]

File: Hosted/Client/wwwroot/index.html (hunk 2 above)
Comment (pranavkm) [reply Hosted/Client/wwwroot/index.html:14]: We updated our templates to no longer use the `app` because it's considered deprecated.

**Pull request closed without merging**